packages/gatsby-theme/src/components/BlogHead/BlogHead.stories.mdx

```diff
@@ -1,6 +1,11 @@
-import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks"
+import { Meta, Story, ArgsTable, Canvas } from "@storybook/addon-docs/blocks"
 import BlogHead from "."
 
+export const author = {
+  name: "Marco José Sánchez",
+  authorLabel: "Marco José",
```

Comment:
Maybe we could change the `authorLabel` prop for just `label`

Reply:
Sure, updated 👍  thanks 🙏